src/UI/templates/default/Button/toggle.less

```diff
@@ -0,0 +1,33 @@
+.il-toggle-button {
+  position: relative;
+  //display: inline-block;
+  height: 19px;
+  padding: 0px 25px;
+  border: 1px solid rgba(0,0,0,0.1);
```

Comment:
Make a less variable. Especially for the color:

"All colors MUST be defined by using variables. In any less file other than variables .less there are therefore only color assignments by directly using a variables allowed "